dashboard/app/models/stage.rb

```diff
@@ -192,8 +192,10 @@ def lockable_state(students)
     end
   end
 
-  # Ensures we get the cached ScriptLevels, vs hitting the db.
+  # Ensures we get the cached ScriptLevels if they are being cached, vs hitting the db.
   def cached_script_levels
+    return script_levels unless Script.should_cache?
```

Comment:
Sure, I understand the need for this line, I'm just wondering if there's a better way we can express things (perhaps by renaming this method? Though I can't think of any good names).